### DISCUSSION TITLE
Github Education application rejected for distance learning student even though the official record states virtual mode

### DISCUSSION THREAD

**Discussion by shobika-a1** [community/community#130896] [GitHub Education]
### Hi everyone,

I am a distance education student from India and trying to apply for github education program. My application is rejected even though my academic letter says I am enrolled to a virtual course. Can someone kindly help me how to proceed?

**Comment by davevad93** [comment]
Hi @shobika-a1 ,

Your academic letter may be missing some necessary information, such as a current date or other data. For instance, my application was rejected when I uploaded my enrollment sheet because it was missing a current date in `DD/MM/YYYY` or `MM/DD/YYYY` format. I finally solved it by uploading a screenshot of my last exam results, however, I'm not a distance learning student.

For more info about rejections, take a look at [Student Developer Pack Application FAQs & Common Rejection Reasons (Updated!)](https://github.com/orgs/community/discussions/111352).
